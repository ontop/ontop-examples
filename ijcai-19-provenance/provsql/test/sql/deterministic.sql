\set ECHO none
set search_path to public, provsql;

CREATE TABLE deterministic_result AS
  SELECT provenance()=provenance()
  FROM personnel
  GROUP BY city;

SELECT remove_provenance('deterministic_result');
SELECT * FROM deterministic_result;
DROP TABLE deterministic_result;  
