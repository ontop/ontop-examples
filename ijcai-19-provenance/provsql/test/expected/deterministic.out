\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

 ?column? 
----------
 t
 t
 t
(3 rows)

